projects/plugins/jetpack/extensions/blocks/contact-form/child-blocks.js

```diff
@@ -0,0 +1,457 @@
+import { createBlock, getBlockType } from '@wordpress/blocks';
```

Comment:
This was just an extraction from `projects/plugins/jetpack/extensions/blocks/contact-form/index.js` to allow the child blocks to be used in the `variations.js`